angular/core/models/discussion_model.coffee

```diff
@@ -128,12 +128,10 @@ angular.module('loomioApp').factory 'DiscussionModel', (DraftableModel, AppConfi
       delete attrs.readSalientItemsCount if attrs.readSalientItemsCount < @readSalientItemsCount
       @baseUpdate(attrs)
 
-    markAsRead: (sequenceId) ->
-      sequenceId = @lastSequenceId if isNaN(sequenceId)
-
-      if @discussionReaderId? and (_.isNull(@lastReadAt) or @lastReadSequenceId < sequenceId)
-        @remote.patchMember @keyOrId(), 'mark_as_read', sequence_id: sequenceId
-        @update(lastReadAt: moment(), lastReadSequenceId: sequenceId)
+    markAsSeen: ->
+      return unless @discussionReaderId and !@lastReadAt
+      @remote.patchMember @keyOrId(), 'mark_as_seen'
```

Comment:
We call mark_as_seen if you land on a thread page, you have a discussion reader, and your lastSeenAt is unpopulated. Mark_as_seen doesn't do much more than `update_attribute` on `last_read_at`